server/migrations/20190617195952-create-private-message.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('PrivateMessages', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    body: {
+      type: Sequelize.STRING,
+      allowNull: false
+    },
+    senderId: {
+      type: Sequelize.INTEGER,
+      allowNull: false
+    },
+    receiverId: {
+      type: Sequelize.INTEGER,
+      allowNull: false,
+      onDelete: 'CASCADE',
+      references: {
+        model: 'Users',
+        key: 'id'
+      }
+    },
+    parentMessageId: {
```

Comment:
Just out of curiosity here - Why do we need the `parentMessageId` really?

Reply:
Okay, I was thinking in a situation where a particular message is sent as a reply to another message. 
`parentMessageId` would be the Id of the message that is being replied to.